examples/dwarfdump.rs

```diff
@@ -896,22 +954,22 @@ fn dump_pubnames<R: Reader>(debug_pubnames: &gimli::DebugPubNames<R>,
                  cu_offset.0,
                  pubname.name().to_string_lossy().unwrap())
```

Comment:
The function is already returning a `Result`, so I would remove the unwrap here too, even though it can't fail for `EndianBuf`. It'll get optimized away, and it makes it easier when looking for other places that unwrap. There might be a few more unwraps like this that you could remove too.